packages/storybook8/stories/Components/ControlBar/Buttons/Participants/index.stories.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ParticipantsButton } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+import { controlsToAdd, hiddenControl } from '../../../../controlsUtils';
+import { ParticipantsButtonDefaultExample } from './snippets/Default.snippet';
+import { ParticipantsButtonWithCallLinkExample } from './snippets/WithCallLink.snippet';
+import { ParticipantsButtonWithCustomRenderExample } from './snippets/WithCustomRender.snippet';
+import { ParticipantsButtonWithCustomStylesExample } from './snippets/WithCustomStyles.snippet';
+import { ParticipantsButtonWithLabelExample } from './snippets/WithLabel.snippet';
+import { ParticipantsButtonWithMuteAllOptionExample } from './snippets/WithMuteAllOption.snippet';
+import { ParticipantsButtonWithUserExcludeFromListExample } from './snippets/WithUserExcludeFromList.snippet';
+
+export { Participants } from './Participants.story';
+
+export const ParticipantsButtonDefaultExampleDocsOnly = {
+  render: ParticipantsButtonDefaultExample
+};
+
+export const ParticipantsButtonWithLabelExampleDocsOnly = {
+  render: ParticipantsButtonWithLabelExample
+};
+
+export const ParticipantsButtonWithCallLinkExampleDocsOnly = {
+  render: ParticipantsButtonWithCallLinkExample
+};
+
+export const ParticipantsButtonWithCustomRenderExampleDocsOnly = {
+  render: ParticipantsButtonWithCustomRenderExample
+};
+
+export const ParticipantsButtonWithCustomStylesExampleDocsOnly = {
+  render: ParticipantsButtonWithCustomStylesExample
+};
+
+export const ParticipantsButtonWithMuteAllOptionExampleDocsOnly = {
+  render: ParticipantsButtonWithMuteAllOptionExample
+};
+
+export const ParticipantsButtonWithUserExcludeFromListExampleDocsOnly = {
+  render: ParticipantsButtonWithUserExcludeFromListExample
+};
+
+const meta: Meta = {
+  title: 'Components/ControlBar/Buttons/Participants',
+  component: ParticipantsButton,
+  argTypes: {
+    isMuteAllAvailable: controlsToAdd.isMuteAllAvailable,
+    showLabel: controlsToAdd.showLabel,
+    callInvitationURL: controlsToAdd.callInvitationURL,
+    participants: controlsToAdd.participantNames,
+    // Hiding auto-generated controls
+    onRenderParticipantList: hiddenControl,
+    styles: hiddenControl,
+    onMuteAll: hiddenControl,
+    strings: hiddenControl,
+    myUserId: hiddenControl,
+    excludeMe: hiddenControl,
+    onRenderParticipant: hiddenControl,
+    onRenderAvatar: hiddenControl,
+    onRemoveParticipant: hiddenControl,
+    onFetchParticipantMenuItems: hiddenControl,
+    showParticipantOverflowTooltip: hiddenControl,
+    onRenderOnIcon: hiddenControl,
+    onRenderOffIcon: hiddenControl,
+    disableTooltip: hiddenControl,
+    tooltipId: hiddenControl,
+    labelKey: hiddenControl
+  },
+  args: {
+    isMuteAllAvailable: false,
+    showLabel: false,
+    callInvitationURL: '',
```

Comment:
Could we remove this line `callInvitationURL `, since there is no default value for it?

Reply:
We need to set default value here with empty string otherwise the control is set to `setString` rather than the text box. Also, we had the default value earlier as well with empty string

<img width="600" alt="Screenshot 2024-07-16 at 12 33 09 PM" src="https://github.com/user-attachments/assets/e61b66ce-97a8-433e-846b-5a103c806662">

<img width="600" alt="Screenshot 2024-07-16 at 12 59 20 PM" src="https://github.com/user-attachments/assets/c0c6c43e-ea18-476f-b5e8-050db8daaea6">

<img width="600" alt="Screenshot 2024-07-16 at 12 59 36 PM" src="https://github.com/user-attachments/assets/81b3ddd1-91ef-44da-a59f-22bed5a78242">